colorls.gemspec

```diff
@@ -14,6 +14,10 @@ ColorLS::POST_INSTALL_MESSAGE = %(
     Help menu     : `colorls -h` provides all possible flag options
 
     Tab completion enabled for flags
+
+    Tree view     : -t has the same effect as --sort=time
+    Report flag   : -r has the same effect as --reverse
```

Comment:
They could use a better more-understandable message. Something like,

```
-t flag : Previously short for --tree, has been re-allocated to sort results by time
-r flag : Previously short for --report, has been re-allocated to reverse sort results
```

Reply:
Yes, that seems better indeed! Done.